packages/emails/src/components/WhenInfo.tsx

```diff
@@ -53,8 +53,10 @@ export function WhenInfo(props: {
     attendee: props.calEvent.attendees[0],
   });
 
+  const isRTL = locale === "he";
+
```

Comment:
Hebru is not the only RTL language; there are more.

Reply:
sure thing! will look up and add another commit to this soon. just need to update an array value I guess.